components/grid/export/overview.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Overview
+page_title: Grid - Export Overview
+description: Export basics for the Grid for Blazor.
+slug: grid-export-overview
+tags: telerik, blazor, grid, export
+published: True
+position: 1
+---
+
+# Blazor Grid Export
+
+The Grid for Blazor provides a built-in functionality to export the data to:
+* [CSV](slug:grid-export-csv)
+* [Excel](slug:grid-export-excel)
+* [PDF](slug:grid-export-pdf)
+
+Before proceeding to the dedicated export articles, ensure you are familiar with the following specifics:
+
+## How the Export Works
+
+The Grid export feature has the following specifics:
+
+* If the Grid is using `OnRead` and is exporting all pages, it will fire an additional `OnRead` event at the time of exporting, with a request `PageSize` of `0`. This will enable the component to obtain all data.
+* The time for export will be longer if:
+    * The Grid has a lot of records and/or 
+    * The Grid is in a Client-side Blazor (WebAssembly) where all the code runs in the browser. At the time of writing, it is considerably slower than server-side Blazor, and it only has one actual thread.
+
+>tip While the file is being generated, the UI will be unresponsive, so you may want to [show a loading sign to the user during the export process](slug:grid-kb-show-loader-while-exporting).
+
+## Requirements
+
+The Grid export feature has the following requirements:
+
+* With Server-side Blazor, the file may become larger than the default SignalR connection limit, and this can disconnect the client and result in an error. Generally, this requires quite a lot of data to happen, but you may need to increase the size limit of the connection in the `ConfigureServices` method of your `Startup.cs` file, for example:
+
+````C#.skip-repl
+services.AddServerSideBlazor().AddHubOptions(o =>
+{
+    o.MaximumReceiveMessageSize = 1024 * 1024; // 1MB
+});
+````
+
+## Limitations
+
+The Grid export feature has the following limitations:
+
+* Templates are not exported, because there is no provision in the framework for getting them at runtime. If a column, header or group header/footer has a template or aggregates, it is be ignored. The headers are the `Title` of the column, the data is the data from the `Field`. If you need additional information, see if you can add it in a Field in the model, or create your own file. Find a <a href="https://feedback.telerik.com/blazor/1485764-customize-the-Pdf-file-before-it-gets-to-the-client" target="_blank">project example on how to generate your own exported file</a>. Find additional information on how to [export an image that is rendered in a Grid column template](slug:grid-export-image-column-excel).
```

Comment:
```suggestion
* Templates are not exported, because there is no provision in the framework for getting `RenderFragment` content at runtime. Thus, column, header or group header/footer templates are ignored. The headers in the exported file match the `Title` of the column. The exported values match the data from the column `Field`. If you need additional information, see if you can add it to a property in the model, or create your own file. Find a [project example on how to generate your own exported file](https://feedback.telerik.com/blazor/1485764-customize-the-Pdf-file-before-it-gets-to-the-client). Find additional information on how to [export an image that is rendered in a Grid column template](slug:grid-export-image-column-excel).
```